### PR TITLE
Update namespace guidance to conform with http guideline.

### PR DESCRIPTION
**Pull request by mbjones**
This addresses issue #223, and pulls text from our decision document on namespace consistency.
It complements the text in the GETTING-STARTED guide, which now says the same thing but explains it
differently. I also considered just creating a forward reference to [GETTING_STARTED](https://github.com/ESIPFed/science-on-schema.org/blob/develop/guides/GETTING-STARTED.md#context), but decided the re-explanation might be useful. Comments and revisions appreciated. You can read the text of the revised section here:

https://github.com/ESIPFed/science-on-schema.org/blob/223-fix-namespace-in-conventions/CONVENTIONS.md#syntax-namespace

### CODE REVIEW

File: CONVENTIONS.md [hunk 1]
```diff
@@ -63,29 +63,57 @@ Figure example:
...
-  ```
-  Including the trailing slash will make the literal representation of terms align with the Internet location of the term definition, and so be clearer for readers and other processors.
-
-  It is further recommended that the prefix `SO:` is used in documentation and other locations when specifically referring to `https://schema.org/`.
-  
+6. **Namespace for `schema.org`.** 
+
+We recommend that the namespace URI for the schema.org vocabulary be consistently set to the value `http://schema.org/`.
+
+ Consistent representation of namespaces simplifies programmatic processing of schema.org markup. For example, even though conceptually it is clear that the terms `http://schema.org/Dataset` and `https://schema.org/Dataset` (note the protocol difference) are referring to [http://schema.org/Dataset](https://schema.org/Dataset), these are programmatically treated as different entities. The [schema.org guidelines](https://schema.org/docs/faq.html#19) are somewhat ambivalent on the topic, but their context files maintain an `http` namespace for all terms, but that can also be resolved from `https`-based addresses.
```
Comment (datadavev) [inline CONVENTIONS.md:70]: Last sentence is a little awkward. Perhaps:

The [schema.org guidelines](https://schema.org/docs/faq.html#19) are somewhat ambivalent on the topic. Their context files maintain an `http` namespace for all terms, but can also be resolved from `https`-based addresses.

**Review by datadavev — APPROVED** [review]
Looks good. One minor, optional minor edit suggestion.

File: CONVENTIONS.md (hunk 1 above)
Comment (mbjones) [reply CONVENTIONS.md]: Sounds good, I added that change.

**Comment by mbjones** [comment]
@ashepherd unless I hear otherwise from you, I'll plan to merge this tomorrow given the feedback from @datadavev. Then we'll be ready for a final review during Monday's meeting.

**Pull request merged**